docs/overview/performance.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Performance
+weight: 8
+---
+
+We track the performance of `django-components` using [ASV](https://asv.readthedocs.io/en/stable/).
+
+See the [benchmarks dashboard](../../benchmarks).
```

Comment:
On the page on performance, we link to the benchmarks dashboard. And the dashboard links back to the doumentation website.